operators/operator_sdk/osdk-new-scorecard.adoc

```diff
@@ -0,0 +1,358 @@
++
+[source,terminal]
+----
+$ operator-sdk alpha scorecard
+----
++
+The scorecard return code is `1` if any of the tests executed did not pass and
+`0` if all selected tests pass.
+
+[id="osdk-new-scorecard-output"]
+== Scorecard output
+
+The `--output` flag specifies the scorecard results output format.
+
+[id="osdk-new-scorecard-output-json"]
+=== JSON format
+
+See an example of the JSON format produced by a scorecard test:
+
+[source,json]
+----
+{
+  "spec": {
+    "image": ""
+  },
+  "status": {
+    "results": [
+      {
+        "name": "olm-bundle-validation",
+        "log": "time=\"2020-06-10T19:02:49Z\" level=debug msg=\"Found manifests directory\" name=bundle-test\ntime=\"2020-06-10T19:02:49Z\" level=debug msg=\"Found metadata directory\" name=bundle-test\ntime=\"2020-06-10T19:02:49Z\" level
+=debug msg=\"Getting mediaType info from manifests directory\" name=bundle-test\ntime=\"2020-06-10T19:02:49Z\" level=info msg=\"Found annotations file\" name=bundle-test\ntime=\"2020-06-10T19:02:49Z\" level=info msg=\"Could not find optio
+nal dependencies file\" name=bundle-test\n",
+        "state": "pass"
+      }
+    ]
+  }
+}
+----
+
+[id="osdk-new-scorecard-output-text"]
+=== Text format
+
+See an example of the text format produced by a scorecard test:
+
+[source,text]
+----
+        Labels:
+        olm-bundle-validation               : pass
+        Log:
+                time="2020-06-10T19:00:43Z" level=debug msg="Found manifests directory" name=bundle-test
+                time="2020-06-10T19:00:43Z" level=debug msg="Found metadata directory" name=bundle-test
+                time="2020-06-10T19:00:43Z" level=debug msg="Getting mediaType info from manifests directory" name=bundle-test
+                time="2020-06-10T19:00:43Z" level=info msg="Found annotations file" name=bundle-test
+                time="2020-06-10T19:00:43Z" level=info msg="Could not find optional dependencies file" name=bundle-test
+----
+
+[NOTE]
+====
+The output format spec matches the link:https://pkg.go.dev/github.com/operator-framework/operator-sdk@v0.19.1/pkg/apis/scorecard/v1alpha3?tab=doc#Test[Test]
+type layout.
+====
+
+[id="osdk-new-scorecard-custom-tests"]
+== Custom tests
+
+Scorecard executes custom tests if they follow these mandated conventions:
+
+* Tests are implemented within a container image.
+* Tests accept an entrypoint which include a command and arguments.
+* Tests produce `v1alpha3` scorecard output in JSON format with no extraneous logging in the test output.
+* Tests can obtain the bundle contents at a shared mount point of `/bundle`.
+* Tests can access the Kubernetes API using an in-cluster client connection.
+
+Writing custom tests in other programming languages is possible if the test
+image follows the above guidelines.
+
+[id="osdk-new-scorecard-complete-config"]
+== Complete scorecard configuration file
+
+[source,yaml]
+----
+tests:
+- name: "basic-check-spec"
```

Comment:
@adellape please change all quay.io/operator-framework/scorecard-test:dev to quay.io/operator-framework/scorecard-test:master